learn/app-development/services/database-services/download-jdbc-driver-jar.md

```diff
@@ -2,12 +2,23 @@
 title: "Download JDBC Driver jar"
 id: ""
 ---
-
+---
 Steps to download the JDBC Driver jar for SQL Server
 
-1. Navigate to [https://www.microsoft.com/en-us/download/details.aspx?id=54671&751be11f-ede8-5a0c-058c-2ee190a24fa6=True&e6b34bbe-475b-1abd-2c51-b5034bcdd6d2=True](https://www.microsoft.com/en-us/download/details.aspx?id=54671&751be11f-ede8-5a0c-058c-2ee190a24fa6=True&e6b34bbe-475b-1abd-2c51-b5034bcdd6d2=True) page, and click **Download JDBC Driver**. **NOTE**:
-    - These download instructions are valid as of the day this document was prepared. Changes made by Microsoft at their website since might not be updated here.
-    - From this site you can download JDBC driver 4.2. You can use JDBC driver 6.0, 6.1 or 6.2, ensure that you import **sqljdbc42.jar** or any Java ver 8 compatible jar file
-2. Select language and click on **Download**
-3. Select and download _sqljdbc\_4.2.8112.100\_enu.tar.gz_
-4. Once the download is complete, extract the downloaded file. Select and upload “sqljdbc42.jar” located : \[your extract folder\] \\sqljdbc\_4.2.8112.100\_enu\\sqljdbc\_4.2\\enu\\jre8 **NOTE**: It is recommended to use **sqljdbc42.jar** or any Java ver 8 compatible jar file. [![](/learn/assets/jdbc_driver_step4.png)](/learn/assets/jdbc_driver_step4.png)
+1. Navigate to [https://www.microsoft.com/en-us/download/details.aspx?id=54671&751be11f-ede8-5a0c-058c-2ee190a24fa6=True&e6b34bbe-475b-1abd-2c51-b5034bcdd6d2=True](https://www.microsoft.com/en-us/download/details.aspx?id=54671&751be11f-ede8-5a0c-058c-2ee190a24fa6=True&e6b34bbe-475b-1abd-2c51-b5034bcdd6d2=True) page, and click **Download JDBC Driver**. 
+
+:::note
+- These download instructions are valid as of the day this document was prepared. Changes made by Microsoft at their website since might not be updated here.
+- From this site you can download JDBC driver 4.2. You can use JDBC driver 6.0, 6.1 or 6.2, ensure that you import **sqljdbc42.jar** or any Java ver 8 compatible jar file.
+:::
+
+2. Select language and click on **Download**.
+3. Select and download `sqljdbc_4.2.8112.100_enu.tar.gz`.
+4. Once the download is complete, extract the downloaded file. Select and upload `sqljdbc42.jar`   
+    Located : [your extract folder] `\sqljdbc_4.2.8112.100_enu\sqljdbc_4.2\enu\jre8` 
+
+:::note
+It is recommended to use **`sqljdbc42.jar`** or any Java ver 8 compatible jar file. 
+:::
+
+[![](/learn/assets/jdbc_driver_step4.png)](/learn/assets/jdbc_driver_step4.png)
```

Comment:
This image url still fails to load

Reply:
Added the image from the old site. This should work now.